XamarinCommunityToolkit/Xamarin.CommunityToolkit.csproj

```diff
@@ -1,7 +1,7 @@
 <?xml version="1.0" encoding="utf-8"?>
 <Project Sdk="MSBuild.Sdk.Extras/2.0.54">
   <PropertyGroup>
-    <TargetFrameworks>netstandard1.0;netstandard2.0;netstandard2.1;Xamarin.iOS10;MonoAndroid90;MonoAndroid10.0;Xamarin.TVOS10;Xamarin.WatchOS10;Xamarin.Mac20;tizen40</TargetFrameworks>
+    <TargetFrameworks>netstandard2.0;netstandard2.1;Xamarin.iOS10;MonoAndroid90;MonoAndroid10.0;Xamarin.TVOS10;Xamarin.WatchOS10;Xamarin.Mac20;tizen40</TargetFrameworks>
```

Comment:
we don't want to drop netstandard1.0 support

Reply:
I added the code to check if image source is valid, but it has APIs which were added in net standard 2.0. I can revert this changes but the issue with Label visibility will remain for net standard 1.0

Reply:
Also why do we use net standard 1.0? In the compatibility table I see that all minimum required OS (android, iOS, Windows) work with net standard 2.0

Reply:
I guess because some of the users can still use PCL